WowPacketParserModule.V6_0_2_19033/Parsers/PetHandler.cs

```diff
@@ -247,5 +247,34 @@ public static void HandlePetActionFeedback(Packet packet)
             packet.ReadInt32<SpellId>("SpellID");
             packet.ReadByteE<PetFeedback>("Response");
         }
+
+        [Parser(Opcode.CMSG_SET_PET_SLOT)]
+        public static void HandleSetPetSlot(Packet packet)
+        {
+            packet.ReadInt32("PetNumber");
+            packet.ReadByte("DestSlot");
+            packet.ReadPackedGuid128("StableMaster");
+        }
+
+        [Parser(Opcode.SMSG_PET_SLOT_UPDATED)]
+        public static void HandlePetSlotUpdated(Packet packet)
+        {
+            packet.ReadInt32("PetNumberA");
+            packet.ReadInt32("PetSlotA");
+            packet.ReadInt32("PetNumberB");
+            packet.ReadInt32("PetSlotB");
+        }
+
+        [Parser(Opcode.SMSG_PET_STABLE_RESULT, ClientVersionBuild.V6_0_2_19033, ClientVersionBuild.V7_3_2_25383)]
```

Comment:
are you sure about 7.3.2 being the version that changed enum?

Reply:
Yes, gonna check later again, when I'm home. But it's definitely not in < 7.x

Reply:
Nevermind. Enum is still the same. Missed a -1 at variable init and didn't notice other changes. Kinda stressed atm - sorry.